examples/utils/example-utils/api-report/example-utils.api.md

```diff
@@ -0,0 +1,316 @@
+## API Report File for "@fluid-example/example-utils"
```

Comment:
There were a number of packages not previously generating API reports. Reviewers will see a number of new reports in this PR.